config/database.yml

```diff
@@ -12,16 +11,15 @@ common: &common
 
 development:
   <<: *common
-
-test: &test
-  <<: *common
-  database: <%= "#{Rails.application.secrets.db_database}-test" %>
+  database: <%= Rails.application.secrets.db_database || "actioncenter_development" %>
```

Comment:
I'd like for this to be actioncenter_development to differentiate from the prod database, but wanted to not break peoples' dev environments so kept previous behavior as default.

Reply:
ok great. I'll do that! had based actioncenter off current db in .env.example, but if we don't care about that anyway then I'll just clear it entirely